packages/studioCMS/src/integrations/studioCMSDashboard/routes/authroutes/login/index.astro

```diff
@@ -82,13 +75,15 @@ if (locals.isLoggedIn) {
 
       <Alert id="form-error" variant="danger" duration={4000} closable>
         <Icon slot="icon" name="exclamation-octagon" />
-        <strong>Something went wrong!</strong><br />
-        <div id = "form-error-message"></div>
+        <strong>Something went wrong!</strong>
+        <Divider />
```

Comment:
So I just want to check that we actually want a divider here, and not just using it to force the element below onto a new line?

the `strong` element could just be `display: block` to give it it's own line

Reply:
For the Alerts, i plan to be using the Divider to add some separation from the prebuilt label, vs the actual response message